app/src/main/java/com/sdpteam/connectout/notifications/NotificationService.java

```diff
@@ -75,10 +74,9 @@ public void createNotificationChannel() {
                 NotificationManager notificationManager = getSystemService(NotificationManager.class);
                 notificationManager.createNotificationChannel(channel);
             }
-        } catch(Exception e) {
+        } catch (Exception e) {
             // do nothing
         }
```

Comment:
Even If this was not written in PR, this approach leaves me perplex, as not all exceptions should be catched here.
What do you think ?

Reply:
I have no idea, i did not worked on this XD